scripts/icu/63.1-min-static-data-1/script.sh

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env bash
+
+# Build ICU common package (libicuuc.a) with data file separate and with support for legacy conversion and break iteration turned off in order to minimize size
+
+MASON_NAME=icu
+MASON_VERSION=63.1-min-static-data-1
```

Comment:
first difference from the `63.1-min-static-data` package is the name change here, adding a `-1` to the end to denote the change.

Reply:
👍